README.md

```diff
@@ -39,12 +39,14 @@ Assist Adnan by writing a program, using any allowed language, to find the city
 
 ### Constraints:
 
-- 1.00 <= price <= 10.00
+- 1.00 <= price <= 100.00
 - we have 101 cities.
 - We have 100 products.
 - Each city may have the same product duplicated several times with different or identical prices.
 - Your code will process a 10^9 (1 billion) line CSV file.
 - You are allowed to code with any language from this list: C, C++, Java, Python, Rust, JavaScript, PHP.
+- The input file is located on "input.txt" 
+- The output file should be located in the same fodler directory as your submission `./output.txt`
```

Comment:
@aboullaite what does this mean?

the output file should mean where the output file will be written by the solution, and it should write the result to a file called `output.txt`. This makes it sound that you have to submit the output file as well, which I don't think what we want

Reply:
How about ?
```suggestion
- Make sure the result are saved in a file saved under `./output.txt`
```

Reply:
yeah looks better (we can even provide a dummy solution that reads from `input.txt` and prints to `output.txt`)